### PR TITLE
Add WHM calls used by TransferTool.

### PR DESCRIPTION
**Pull request by ruyrocha**

### CODE REVIEW

**Comment by fabiokr** [comment]
:+1:

File: lib/lumberg/whm/transfer_tool.rb [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                            transfer_threads: 2,
+                            restore_threads: 2,
+                            unrestricted_restore: 1,
+                            compressed: 1,
+                            unencrypted: 0,
+                            use_backups: 1,
+                            low_priority: 0,
+                            enable_custom_pkgacct: 0 }
+
+        options.merge!({ "api.version" => "1" })
```
Comment (itspriddle) [inline lib/lumberg/whm/transfer_tool.rb]: Can you remove the curly braces here and in the other `merge!` calls below? We typically skip them in method calls like this.

**Comment by rubemz** [comment]
:+1: besides josh's comment

**Comment by rubemz** [comment]
@boone ah ok

**Pull request merged**